fixtures/features/api-elements/extensions.json

```diff
@@ -14,6 +14,26 @@
       "attributes": {},
       "content": [
         {
+          "element": "extension",
+          "meta": {
+            "links": [
+              {
+                "element": "link",
+                "meta": {},
+                "attributes": {
+                  "relation": "profile",
+                  "href": "http://swagger.io/specification/#vendorExtensions"
```

Comment:
I am wondering whether the uri here is the right thing to do. Should we be using some other link in docs.apiary.io domain?

I think @smizell should look at this and answer this.